src/domain/customers/CustomerListComponent.tsx

```diff
@@ -0,0 +1,97 @@
+import React from 'react';
+
+import Table from '../../common/table/Table';
+import CustomerDetails from './CustomerDetailsComponent';
+
+export interface CustomerData {
+  comment?: string;
+  contactMethod?: string;
+  email?: string;
+  firstName?: string;
+  id?: string;
+  image?: string;
+  invoicingType?: string;
+  language?: string;
+  lastName?: string;
+  nickname?: string;
+  phone?: string;
+}
+
+interface TableData {
+  goToDetails: string;
+  group: string;
+  invoice: string;
+  name: string;
+  queue: string;
+  startDate: string;
+  thing: string;
+}
+
+interface Props {
+  data: CustomerData | null | any;
+}
+
+const HarborsListComponent = ({ data }: Props) => {
+  const columns = [
+    {
+      Header: 'Jono',
+      accessor: 'queue',
+    },
+    {
+      Header: 'Asiakas',
+      accessor: 'name',
+    },
+    {
+      Header: 'Liittynyt',
+      accessor: 'startDate',
+    },
+    {
+      Header: 'Asiakasryhmä',
+      accessor: 'group',
+    },
+    {
+      Header: 'Asia',
+      accessor: 'thing',
+    },
+    {
+      Header: 'Laskuja',
+      accessor: 'invoice',
+    },
+    {
+      Header: 'Avaa',
+      accessor: 'goToDetails',
+    },
```

Comment:
Header's value should be added to `fi.json`and retrieved from it.

Reply:
fixed